src/app/convenience-file-types.js

```diff
@@ -1,8 +1,175 @@
+import { FileType } from './constants';
+
+function getAnndataBaseFileDef(fileDef) {
+  return {
+    url: fileDef.url,
+    requestInit: fileDef.requestInit,
+    coordinationValues: {
+      ...fileDef.coordinationValues,
+      obsType: fileDef.coordinationValues?.obsType || 'cell',
+      featureType: fileDef.coordinationValues?.featureType || 'gene',
+    },
+  };
+}
+
+export function expandAnndataCellsZarr(fileDef) {
+  const baseFileDef = getAnndataBaseFileDef(fileDef);
+  const { options = {} } = fileDef;
+  const embeddingTypes = options.mappings ? Object.keys(options.mappings) : [];
+  return [
+    {
+      ...baseFileDef,
+      fileType: FileType.OBS_INDEX_ANNDATA_ZARR,
+      options: {
+        path: 'obs/index',
+      },
+    },
```

Comment:
Do we need to check that this is not being fed twice identically if there is also an `expression-matrix`, say?  Or would that not break things?